src/js/utils/utils.js

```diff
@@ -9,8 +9,12 @@ const {
   version,
   name,
 } = require('../../../package.json');
+
 const Logger = require('./logger');
 
+// @todo - remove this side-effect
```

Comment:
it's a side effect because logic is executed when the file is required/imported - i can update the comment to say that, but i think i'd just be defining "side-effect"